src/pages/FileSystem.tsx

```diff
@@ -65,8 +65,27 @@ export const FileSystemPage = () => {
                 event.stopPropagation();
             }}
             className={selectedStorage !== undefined ? "dialog-btn" : "dialog-btn disabled"}
-            style={{ float: "left", width: "100%" }}
-        >Load crew from local storage</button>
+            style={{ float: "left", width: "45%" }}
+        >Load crew</button>
+        <button
+            onClick={(event) => {
+                event.preventDefault();
+                event.stopPropagation();
+                if (selectedStorage) {
+                    const key = `${selectedStorage.ShipName}_${selectedStorage.Captain?.name}_${selectedStorage.Captain?.level}_${selectedStorage.Credits}`;
+                    localStorage.removeItem(key);
+                    // Not the finest craft, but to make this proper,
```

Comment:
we can solve this with react hooks. window reload is a bit much for this
i'll prepare a new commit